ProjectBank/Client/Pages/Index.razor

```diff
@@ -2,6 +2,41 @@
 
 <PageTitle>Index</PageTitle>
 
-<h1>Hello, world!</h1>
+@if (yourProjects.Count > 0){
+    <h1>Your Projects</h1>
+    <div class="row justify-content-around">
+        @foreach (var project in yourProjects)
+        {
+            <div class="card col col-xs-12 col-sm-6 col-md-4 col-lg-3" style="width: 18rem;">
+                <div class="card-body">
+                    <h5 class="card-title">@project.Title</h5>
+                    <p class="card-text">@project.Description</p>
+                    <a href="project/@project.Id/" class="btn btn-primary stretched-link">View Project</a>
+                </div>
+            </div>
+        }
+    </div>
+}
+<h1>Explore</h1>
+<div class="row justify-content-around">
+    @foreach (var project in projectsList)
+    {
+        <div class="card mt-4 col col-xs-12 col-sm-6 col-md-4 col-lg-3" style="width: 18rem;">
+            <div class="card-body">
+                <h5 class="card-title">@project.Title</h5>
+                <p class="card-text">@project.Description</p>
+                <a href="project/@project.Id/" class="btn btn-primary stretched-link">View Project</a>
+            </div>
+        </div>
+    }
+</div>
 
-Welcome to your new app.
+@code {
+    List<Project> yourProjects = new List<Project>();
```

Comment:
Looks great :)

Later on we have to remember to change the Lists to holding ProjectDetailedDTO instead.